modules/about/con_understanding-internal-developer-platforms.adoc

```diff
@@ -0,0 +1,72 @@
+:_newdoc-version: 2.18.3
+:_template-generated: 2025-05-12
+
+:_mod-docs-content-type: CONCEPT
+
+[id="understanding-internal-developer-platforms_{context}"]
+= Understanding internal developer platforms
+
+An internal developer platform (IDP) is a curated set of tools and services that supports developer self-service. Instead of navigating multiple systems, developers use a unified interface to provision environments, deploy code, and access APIs.
+
+Why IDPs matter::
+
+IDPs address the challenges of modern software delivery by enabling self-service, enforcing standards, and improving the developer experience.
+
+For developers::
+
+* *Faster onboarding:* Developers can use learning paths, software templates, and the software catalog to deploy compliant services within minutes—without depending on other teams for setup.
+* *Reduced cognitive load:* Tools, documentation, and deployment environments are available in one place, eliminating the need to switch between systems or manage disconnected resources.
+* *Self-service workflows:* Developers can create applications or environments on demand, without raising tickets or waiting for approvals.
+* *Built-in standards:* Platform-defined templates and software templates ensure secure and compliant workflows by default.
+* *Cross-team visibility:* Shared service catalogs and documentation improve discoverability and reuse across teams.
+* *Higher productivity:* Developers spend more time building features and less time configuring infrastructure or resolving toolchain inconsistencies.
+
+For platform engineers::
```

Comment:
I was thinking the organizations section might come first followed by platform engineers and developers section. wdyt?